api/src/test/java/uk/gov/dhsc/htbhf/claimant/entitlement/PregnancyEntitlementCalculatorTest.java

```diff
@@ -74,4 +83,36 @@ void shouldThrowExceptionWhenEntitlementDateIsNull() {
 
         assertThat(thrown.getMessage()).isEqualTo("entitlementDate must not be null");
     }
+
+    @Test
+    void shouldReturnTrueWhenPaymentCycleIsSecondToLastOneWithPregnancyVouchers() {
+        // The claimant will receive pregnancy vouchers for this cycle and the one after but not after that (given a 12 week grace period and four week cycles).
+        LocalDate expectedDeliveryDate = LocalDate.now().minusWeeks(5);
+        Claim claim = aClaimWithExpectedDeliveryDate(expectedDeliveryDate);
+        PaymentCycle paymentCycle = aPaymentCycleWithClaim(claim);
+
+        boolean result = calculator.currentCycleIsClaimantsSecondToLastCycleWithPregnancyVouchers(paymentCycle);
+
+        assertThat(result).isTrue();
+    }
+
+    @ParameterizedTest
+    @MethodSource("provideArgumentsForShouldReturnFalseWhenPaymentCycleIsNotSecondToLastOneWithPregnancyVouchers")
+    void shouldReturnFalseWhenPaymentCycleIsNotSecondToLastOneWithPregnancyVouchers(LocalDate expectedDeliveryDate) {
+        Claim claim = aClaimWithExpectedDeliveryDate(expectedDeliveryDate);
+        PaymentCycle paymentCycle = aPaymentCycleWithClaim(claim);
+
+        boolean result = calculator.currentCycleIsClaimantsSecondToLastCycleWithPregnancyVouchers(paymentCycle);
+
+        assertThat(result).isFalse();
+    }
+
+    private static Stream<Arguments> provideArgumentsForShouldReturnFalseWhenPaymentCycleIsNotSecondToLastOneWithPregnancyVouchers() {
```

Comment:
Suggest calling this something like `expectedDeliveryDatesNotRequiringReminderEmail` or similar

Reply:
Renamed.